Microsoft.Toolkit.Uwp.UI.Animations/Behaviors/ViewportBehavior.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Toolkit.Uwp.UI.Extensions;
+using Microsoft.Xaml.Interactivity;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Animations.Behaviors
+{
+    /// <summary>
+    /// A class for listening element enter or exit the ScrollViewer viewport
+    /// </summary>
+    public class ViewportBehavior : BehaviorBase<FrameworkElement>
+    {
+        /// <summary>
+        /// The ScrollViewer hosting this element.
+        /// </summary>
+        private ScrollViewer _hostScrollViewer;
+
+        /// <summary>
+        /// The IsFullyInViewport value of the associated element
+        /// </summary>
+        public static readonly DependencyProperty IsFullyInViewportProperty =
+            DependencyProperty.Register(nameof(IsFullyInViewport), typeof(bool), typeof(ViewportBehavior), new PropertyMetadata(default(bool), OnIsFullyInViewportChanged));
+
+        /// <summary>
+        /// The IsInViewport value of the associated element
+        /// </summary>
+        public static readonly DependencyProperty IsInViewportProperty =
+            DependencyProperty.Register(nameof(IsInViewport), typeof(bool), typeof(ViewportBehavior), new PropertyMetadata(default(bool), OnIsInViewportChanged));
+
+        /// <summary>
+        /// The IsRemoveBehaviorAfterEnteredViewport value of the associated element
+        /// </summary>
+        public static readonly DependencyProperty IsRemoveBehaviorAfterEnteredViewportProperty =
```

Comment:
Can we invert the logic for this property and rename it to `IsAlwaysOn`? (default false)

Reply:
Done.